src/PAModel/Serializers/SourceSerializer.cs

```diff
@@ -321,10 +321,24 @@ private static void LoadTemplateFiles(ErrorContainer errors, CanvasDocument app,
                 templateList.Add(new TemplatesJson.TemplateJson() { Name = templateName, Template = xmlContents, Version = parsedTemplate.Version });
             }
 
+            List<PcfTemplateJson> pcfTemplateConversionsList = null;
+            string pcfTemplatePath = Path.Combine(packagesPath, PCFConversionFileName);
+            if (File.Exists(pcfTemplatePath))
+            {
+                pcfTemplateConversionsList = new List<PcfTemplateJson>();
+
+                DirectoryReader.Entry file = new DirectoryReader.Entry(pcfTemplatePath);
+                var pcfTemplateConversions = file.ToObject<PcfTemplateJson[]>();
```

Comment:
>pcfTemplateConversions [](http://example.com/codeflow?start=20&length=22)

What's the difference between this and pcfTemplateConversionsList?

Reply:
Thanks for catching this. Was handling the array differently earlier while working, now we don't need pcfTemplateConversionsList.